src/app/Dice.elm

```diff
@@ -0,0 +1,19 @@
+module Dice exposing (emptyCup, Cup, cupFromIntList)
+
+
+type alias Die =
+    Int
+
+
+type alias Cup =
+    List Die
+
+
+emptyCup : Cup
+emptyCup =
+    []
+
+
+cupFromIntList : List Int -> Cup
+cupFromIntList =
+    identity
```

Comment:
This should be called `init : List Int -> Cup` as it will make constructors across modules uniform. `Cup.init`, `Game.init`, `Socket.init` etc... and it's not a good idea to put the current set of parameter types in the function name for a constructor.